packages/hooks/src/hooks/useEffectAfterInitialRender.ts

```diff
@@ -0,0 +1,22 @@
+import { EffectCallback, useEffect, useRef } from "react";
+
+/**
+ * Like React's useEffect, except doesn't execute on initial render.
+ */
+export const useEffectAfterInitialRender: typeof useEffect = (
+  effect,
+  deps?
+) => {
+  const didMount = useRef(false);
+  let cleanup: ReturnType<EffectCallback> = undefined;
+
+  useEffect(() => {
+    if (didMount.current) {
+      cleanup = effect();
```

Comment:
I think the cleanup needs to be returned from the effect callback here, as otherwise React won't be calling it appropriately. See https://reactjs.org/docs/hooks-reference.html#cleaning-up-an-effect

Reply:
Changed.